scripts/Docker/Dockerfile

```diff
@@ -41,6 +41,7 @@ RUN ln -s /usr/src/librealsense-$LIBRS_VERSION /usr/src/librealsense
 RUN cd /usr/src/librealsense \
     && mkdir build && cd build \
     && cmake \
+    -DPYTHON_EXECUTABLE=/usr/bin/python3 \
```

Comment:
Is it possible to use `which python3`? Will help if there are several versions of python3 installed